docker.setup.js

```diff
@@ -0,0 +1,273 @@
+// Copyright IBM Corp. 2017,2019. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+const _ = require('lodash');
+const async = require('async');
+const spawn = require('child_process').spawn;
+const docker = new require('dockerode')();
+const fmt = require('util').format;
+const http = require('http');
+const ms = require('ms');
+const fs = require('fs-extra');
+
+// we don't pass any node flags, so we can call _mocha instead the wrapper
+const mochaBin = require.resolve('mocha/bin/_mocha');
+
+process.env.CLOUDANT_DATABASE = 'test-db';
```

Comment:
Can we change this to `testdb`? Most of our default db names are `testdb` :smile:

Reply:
@agnes512 sure whatever name makes more sense to lb-next tests~